CRM/Upgrade/Incremental/sql/5.15.alpha1.mysql.tpl

```diff
@@ -9,3 +9,24 @@ SELECT @option_group_id_ps as option_group_id, {localize field='label'}`label`{/
 FROM civicrm_option_value ov
 INNER JOIN civicrm_option_group og
 ON og.id = ov.option_group_id AND og.name = 'contribution_status';
+
+SELECT @maxValue := MAX(value) FROM `civicrm_option_value` where option_group_id = @option_group_id_ps;
+SELECT @maxWeight := MAX(weight) FROM `civicrm_option_value` where option_group_id = @option_group_id_ps;
+
+INSERT INTO `civicrm_option_value` (
+`option_group_id`, {localize field='label'}`label`{/localize}, `value`, `name`, `weight`, `is_reserved`, `is_active`, `is_default`
+)
+VALUES(
+@option_group_id_ps, 'Processing', @maxValue + 1, 'Processing', @maxWeight + 1, 1 , 1 , 0
+)
+ON og.id = ov.option_group_id AND og.name = 'contribution_status'
+ON DUPLICATE KEY UPDATE id=id;
```

Comment:
How about using INSERT IGNORE statement over ON DUPLICATE KEY, since the statement is not doing any updates.

Reply:
so when I googled the 2 were functionally equivalent but I have a niggle at the back of my mind that in the past we got caught out on INSERT IGNORE ignoring more serious errors than we wanted it too